packages/now-cli/test/fixtures/unit/now-dev-static-build-routing/now.json

```diff
@@ -3,7 +3,7 @@
   "builds": [
     {
       "src": "package.json",
-      "use": "@now/static-build",
+      "use": "@now/static-build@canary",
```

Comment:
Why canary?

Reply:
Because of https://github.com/zeit/now/pull/3136 and https://github.com/zeit/now/pull/3137. If those have been released on stable then we can revert this part.

Reply:
Interesting. I think we need a better approach now that we have a monorepo 🤔 

Ideally, we should be able to run `now dev` tests against builders without publishing to npm.